generator/internal/service_code_generator.cc

```diff
@@ -121,6 +141,26 @@ bool ServiceCodeGenerator::HasStreamingReadMethod() const {
                      });
 }
 
+std::vector<std::string>
+ServiceCodeGenerator::MethodSignatureWellKnownProtobufTypeIncludes() const {
+  std::vector<std::string> include_paths;
+  for (auto method : methods_) {
+    auto method_signature_extension =
+        method.get().options().GetRepeatedExtension(
+            google::api::method_signature);
+    google::protobuf::Descriptor const* input_type = method.get().input_type();
+    for (auto const& extension : method_signature_extension) {
+      std::vector<std::string> parameters = absl::StrSplit(extension, ",");
```

Comment:
When `StrSplit`ing w/ single characters, use single quotes to make the object a `char`, which will be slightly more efficient. I, personally, don't care much, but when we move to clang-tidy 13 (or when we import the code), there will be a warning: https://clang.llvm.org/extra/clang-tidy/checks/abseil-faster-strsplit-delimiter.html